apps/user-profile/src/assets/i18n/en.json

```
@@ -168,6 +169,12 @@
     "DELETE_IMG_DIALOG_DESC": "Your security image will be deleted and will not be used during authentication process.",
     "MFA_INFO": "Manage my MFA tokens"
   },
+  "LOCAL_ACCOUNT": {
+    "TITLE": "Local account",
+    "DESC": "Manage your account in namespace",
```

```suggestion
    "DESC": "Manage your local account",
```
After the discussion with Peter, we wanted this text to be shown to the user without the information of the name of the namespace (user don't have to see name of the namespace, why would he?).
Dont forget to change the Czech translate